examples/aws_flows/variables.tf

```diff
@@ -0,0 +1,37 @@
+variable "flow_logs_bucket_arns" {
+  type        = list(string)
+  description = "List of S3 buckets having flow logs"
+  validation {
+    condition     = length(var.flow_logs_bucket_arns) > 0
+    error_message = "The flow_logs_bucket_arns value must not be empty."
+  }
+}
+
```

Comment:
Remove this. Give a concrete list of literal ARNs in the example to illustrate how to use the module.